orgbee/src/app/pages/dashboard/settings/settings.component.ts

```diff
@@ -0,0 +1,368 @@
+import { Component, OnInit, Input, EventEmitter, Output } from '@angular/core';
+import { FormGroup, FormBuilder, Validators, AbstractControlOptions, 
+  ValidatorFn, AbstractControl, FormControl,
+  ValidationErrors } from '@angular/forms';
+import { catchError } from 'rxjs/operators';
+import { of } from 'rxjs';
+
+import { MustMatch } from './confirmed.validator';
+import { LoginService, UserInfo } from '../../../../service/login-service/login.service';
+import { UserService } from '../../../../service/user-service/user.service';
+import { Router } from '@angular/router';
+import { ConfirmationDialogService } from '../../../../service/confirmation-dialog-service/confirmation-dialog.service';
+import { ResponseService } from '../../../../service/response-service/response.service';
+import { SpinnerService } from '../../../../service/spinner-service/spinner.service';
+
+@Component({
+  selector: 'app-settings',
+  templateUrl: './settings.component.html',
+  styleUrls: ['./settings.component.css']
+})
+export class SettingsComponent implements OnInit {
+  updateForm: FormGroup;
+  passwordForm: FormGroup;
+  deleteForm: FormGroup;
+  showInformation = false;
+  showAccountManagement = false;
+  showAccountDeletion = false;
+  showDeactivateModal = false;
+  showDeleteModal = false;
+  showChangeAvatar = true;
+  changesMade: boolean = false;
+  userInfo: UserInfo;
+  selectedAvatarPath: string = ''; 
+  currentSelectedAvatar: string = ''; 
+  selectedTab: string = 'profile';
+
+  constructor(
+    private formBuilder: FormBuilder, 
+    private loginService: LoginService,
+    private userService: UserService,
+    private router: Router,
+    private responseService: ResponseService,
+    private confirmationDialogService: ConfirmationDialogService,
+    private spinnerService: SpinnerService
+  ) {}
+
+  @Input() showSettings: boolean = false;
+  @Output() close: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
In creating elements, class components, set the format to:
```
export class ClassName {
  @Input properties;

  @Output properties;

  @ViewChild ElementRef;

  classProperties;

  observables;

  constructor(){}

  lifecycleMethods(){}

  userDefinedMethods(){}
}

```

See the link for reference. 
https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g2ce365fc0ac_0_5